test/libsolidity/util/SoltestTypes.h

```diff
@@ -139,6 +139,7 @@ struct ABIType
 struct FormatInfo
 {
 	bool newline = false;
+	bool hide = false;
```

Comment:
This could be used to hide parameters from the output e.g.:
```
// e() -> FAILURE, hex"08c379a0", 0x20, 19, "Transaction failed."
```
becomes
```
// e() -> FAILURE, "Transaction failed."
```
Even though we decided to not hide the signature, the tail-pointer and the length (see discussion), I left it in since it could be useful in the future.

Reply:
Maybe add this as comment in the code too? :)

Reply:
I would actually prefer to remove that functionality, if it is off by default.

Reply:
I agree and removed it.